docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/custom-build.md

```diff
@@ -58,29 +64,35 @@ Configuring pre-defined devices will allow the service to automatically provisio
 
 1. Make sure `secret name` is set to match `SecretName` in `camera.yaml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
 
+      
 1. Under `secretName`, set `username` and `password` to your camera credentials. If you have multiple cameras copy the `Writable.InsecureSecrets` section and edit to include the new information.
 
-   ```yaml
-   Writable:
-   LogLevel: INFO
-   InsecureSecrets:
-      credentials001:
-         SecretName: credentials001
-         SecretData:
-         username: <Credentials 1 username>
-         password: <Credentials 1 password>
-         mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
-      credentials002:
-         SecretName: credentials002
-         SecretData:
-         username: <Credentials 2 username>
-         password: <Credentials 2 password>
-         mode: usernametoken    # assign "digest" | "usernametoken" | "both" | "none"
-   ```
+      <div class='admonition warning'>
+         <p class='admonition-title'>Warning</p>
+         <p>Be careful when storing any potentially important information in cleartext on files in your computer. In this case, the credentials for the camera(s) are stored in cleartext in the `configuration.yaml` file on your system. </p>
```

Comment:
I would add something about `InsecureSecrets` is for non-production use since the credentials are stored in plain text